src/sentry/replays/lib/summarize.py

```diff
@@ -232,23 +266,39 @@ def generate_summary_logs(
                 error_idx < len(error_events) and error_events[error_idx]["timestamp"] < timestamp
             ):
                 error = error_events[error_idx]
-                yield generate_error_log_message(error)
+
+                if error["category"] == "error":
+                    yield generate_error_log_message(error)
+                elif error["category"] == "feedback" and error["id"] not in seen_feedback_ids:
```

Comment:
```suggestion
                elif error["category"] == "feedback":
```
Trace connected query results are unique